Examples/Io/Root/src/RootSimHitWriter.cpp

```diff
@@ -70,23 +70,17 @@ ActsExamples::RootSimHitWriter::~RootSimHitWriter() {
 }
 
 ActsExamples::ProcessCode ActsExamples::RootSimHitWriter::endRun() {
-  if (m_outputFile != nullptr) {
-    m_outputFile->cd();
-    m_outputTree->Write();
-    ACTS_VERBOSE("Wrote hits to tree '" << m_cfg.treeName << "' in '"
-                                        << m_cfg.filePath << "'");
-    m_outputFile->Close();
-  }
+  m_outputFile->cd();
+  m_outputTree->Write();
+  ACTS_VERBOSE("Wrote hits to tree '" << m_cfg.treeName << "' in '"
+                                      << m_cfg.filePath << "'");
+  m_outputFile->Close();
```

Comment:
```suggestion
  m_outputFile->cd();
  m_outputTree->Write();
  m_outputFile->Close();
  ACTS_VERBOSE("Wrote hits to tree '" << m_cfg.treeName << "' in '"
                                      << m_cfg.filePath << "'");
```

We have here and at the other positions different orderings of `ACTS_VERBOSE` and `->Close()`. Should we decide for one?

Reply:
good idea :+1: